docs/modules/idrac_secure_boot.rst

```diff
@@ -0,0 +1,207 @@
+.. _idrac_secure_boot_module:
+
+
+idrac_secure_boot -- Import secure boot certificate.
+====================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Synopsis
+--------
+
+This module allows to import the secure boot certificate.
+
+
+
+Requirements
+------------
+The below requirements are needed on the host that executes this module.
+
+- python \>= 3.9.6
+
+
+
+Parameters
+----------
+
+  import_certificates (optional, bool, None)
+    Import all the specified key certificates.
+
+    When \ :emphasis:`import\_certificates`\  is \ :literal:`true`\  either of \ :emphasis:`platform\_key`\  or \ :emphasis:`KEK`\  or \ :emphasis:`database`\  or \ :emphasis:`disallow\_database`\  is required.
+
+
+  platform_key (optional, path, None)
+    The absolute path of the Platform key certificate file for UEFI secure boot.
+
+
+  KEK (optional, list, None)
+    A list of absolute paths of the Key Exchange Key (KEK) certificate file for UEFI secure boot.
+
+
+  database (optional, list, None)
+    A list of absolute paths of the Database certificate file for UEFI secure boot.
+
+
+  disallow_database (optional, list, None)
+    A list of absolute paths of the Disallow Database certificate file for UEFI secure boot.
+
+
+  restart (optional, bool, False)
+    Secure boot certificate import operation requires a server restart. This parameter provides an option to restart the server.
+
+    \ :literal:`true`\  will restart the server.
```

Comment:
restarts the server.

does not restart the server.

Reply:
- C(true) restarts the server.
- C(false) does not restart the server.